metricbeat/module/pgbouncer/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.9'
+
+services:
+  postgresql:
+    image: postgres:15
+    hostname: postgresql
+    environment:
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_USER=test
+      - POSTGRES_DB=test
+      - POSTGRES_HOST_AUTH_METHOD=md5
+      - POSTGRES_INITDB_ARGS=--auth=md5
+
+  pgbouncer:
+    image: edoburu/pgbouncer
+    hostname: pgbouncer
+    ports:
+      - 6432:6432
+    volumes:
+      - ./_meta/pgbouncer.ini:/etc/pgbouncer/pgbouncer.ini
+      - ./_meta/userlist.txt:/etc/pgbouncer/userlist.txt
+    healthcheck:
+      test: ['CMD-SHELL', 'psql -h localhost -p 6432 -U test -d pgbouncer -c "SHOW STATS"']
+      interval: 15s
+      timeout: 30s
+      retries: 5
+      start_period: 15s
+
+  elasticsearch:
+    image: elasticsearch:8.11.0
```

Comment:
Probably better to use image version from https://github.com/elastic/beats/tree/main/testing/environments

Reply:
[c72880a](https://github.com/elastic/beats/pull/41174/commits/c72880a60f76a0c27c8bb547b2c83e30f5b12217)